src/components/MovieCard.js

```diff
@@ -1,19 +1,71 @@
 import "./MovieCard.css";
-import React from "react";
+import React, {useEffect, useRef, useState, Suspense} from "react";
 import PropTypes from "prop-types";
 
-const  MovieCard = ({movie}) => (
-    <div className="movie-card">
-        <img src={require(`../static/images/movies/${movie.icon}`)}/>
-        <div className="movie-card-info">
-            <div className="movie-card-name">{movie.name}</div>
-            <div className="movie-card-genres">
-                {movie.genres.map(genre=>genre.name).join(", ")}
+const ContextMenu = React.lazy(()=> import("./ContextMenu"));
+
+
+const  MovieCard = function(props){
```

Comment:
don't afraid to use arrow functions
```suggestion
const  MovieCard = (props) => {
```

Reply:
done